### ISSUE TITLE
xt_dscp.ko.xz already exists

### ISSUE THREAD

**Issue by amencarini** [nerves-project/nerves_system_rpi3#305]
### Environment

* Elixir version (`elixir -v`): Elixir 1.15.4 (compiled with Erlang/OTP 26)
* Nerves environment: (`mix nerves.env --info`)
  MIX_TARGET:   rpi3
  MIX_ENV:      dev

  nerves_system_rpi3 version 1.25.0

* Additional information about your host, target hardware or environment that may help
  * Host is macOS 14.1.2

### Current behavior

Include errors, stacktraces, screenshots or code that may help us reproduce the
issue.

When running `mix firmware` on a clean install I get the following:

```
==> nerves
==> device

Nerves environment
  MIX_TARGET:   rpi3
  MIX_ENV:      dev

Generated device app
|nerves| Building OTP Release...

* [Nerves] validating vm.args
* skipping runtime configuration (config/runtime.exs not found)
* creating _build/rpi3_dev/rel/device/releases/0.1.0/vm.args
Updating base firmware image with Erlang release...
Copying rootfs_overlay: /Users/ale/Projects/elixir/hermus_new/device/_build/rpi3_dev/nerves/rootfs_overlay
Copying rootfs_overlay: /Users/ale/Projects/elixir/hermus_new/device/rootfs_overlay

FATAL ERROR: write_file: file squashfs-root/lib/modules/6.1.63-v7/kernel/net/netfilter/xt_dscp.ko.xz already exists
```

### Expected behavior

Not receiving the error

**Comment by fhunleth** [comment]
Thanks for the report. We made a mistake when releasing the latest version of Nerves System RPi3. Older versions work. This latest one has an update to Linux 6.1 that is missing many of the changes we normally make.

I started a branch at https://github.com/nerves-project/nerves_system_rpi3/pull/306.

Unfortunately, I don't have a RPi 3 with me right now to test it out. It will be a lot closer than what's in `main` and will fix the problem that you ran into. You may run into another problem since I didn't test it, but I'm more optimistic.

**Comment by amencarini** [comment]
Let me know if there's a way to test out your branch, happy to test drive it if there's a set of instructions I can follow (I'm very new to Nerves!)

**Comment by fhunleth** [comment]
Hmm. I'd love help, but honestly, we really try to minimize the amount of time people deal with the Linux/Buildroot pieces with Nerves. I don't want to inflict that upon you while you're new. This part requires a decent amount of Linux debugging if it doesn't work. I don't think it will be long before we have a fix out. The biggest delay will be that the Nerves core team isn't around that much right now over the holidays.

I'd recommend trying a different SBC if you have one. Of the Raspberry Pis, we use Zero, Zero 2, and 4B the most these days. The Zeros are particularly convenient since you can just plug them in via USB to your laptop. The Beaglebones are pretty frequently used as well.

**Comment by fhunleth** [comment]
This is fixed in 1.25.1. Thanks for reporting!
